tensorboard/pip_package/setup.py

```diff
@@ -25,6 +25,8 @@
 
 
 REQUIRED_PACKAGES = [
+    'futures >= 3.1.1',
+    'grpcio >= 1.4.0',
```

Comment:
grpcio is fine as a transitive dependency of tensorflow. tensorflow is already using grpc (in its distributed_runtime, through C++). the size of grpcio is not excessive (at 4-5 MB), as compared with other dependency pip packages of tensorflow, e.g., numpy (15 MB) and protobuf (5 MB).